api/CcsSso.Core.Service/OrganisationService.cs

```diff
@@ -354,14 +355,32 @@ private async Task CreateOrganisationAsync(OrganisationRegistrationDto organisat
         };
 
         await _organisationProfileService.CreateOrganisationAsync(organisationProfileInfo);
-      }
+        await OrganisationAdminGroup(organisationRegistrationDto, ciiOrgId);
+        }
       catch (Exception)
       {
         await _ciiService.DeleteOrgAsync(ciiOrgId);
         throw;
       }
     }
+        public async Task OrganisationAdminGroup(OrganisationRegistrationDto organisationRegistrationDto, string ciiOrgId)
+        {
+            var Organisation = await _dataContext.Organisation.Include(o => o.UserGroups).FirstOrDefaultAsync(o => !o.IsDeleted && o.CiiOrganisationId == ciiOrgId);
+            var IsDefaultAdminGroup = await _dataContext.OrganisationUserGroup.AnyAsync(x => x.OrganisationId == Organisation.Id && x.GroupType == (int)GroupType.Admin && !x.IsDeleted);
 
+            if (!IsDefaultAdminGroup)
+            {
+                var group = new OrganisationUserGroup
+                {
+                    OrganisationId = Organisation.Id,
+                    UserGroupName = "Organisation Administrators",
+                    UserGroupNameKey = "ORGANISATION_ADMINISTRATORS",
```

Comment:
move group name and key to constant file.

Reply:
Moved group name and key to constant file.